platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx

```diff
@@ -170,23 +205,3 @@ See this example in the configuration with `data-studios-rsa.pem` file mounted i
     volumes:
       - $PWD/data-studios-rsa.pem:/data-studios-rsa.pem
```

Comment:
Parking? aka TBD?

Reply:
@gwright99, I wasn't ready to delete the original content that was merged in regards to the Docker Compose installation approach, but this entire section will go away.